### ISSUE TITLE
Label not showing

### ISSUE THREAD

**Issue by ozdemiremre** [omsharp/BetterComments#6]
### Installed product versions
- Visual Studio: 2015 Enterprise
- This extension's version: 1.1.9

### Description
Settings menu doesn't have labels. Can't see the text.

### Steps to recreate
1. Open the settings dialog from Tools-> Options-> Better Comments

### Current behavior
Doesn't show the labels' texts.

### Expected behavior
Dialog should show what the text are.

![Dailog](https://cloud.githubusercontent.com/assets/8995420/16544911/d0e626c0-411f-11e6-9385-2baedc6bfac6.PNG)

**Comment by omsharp** [comment]
Does it happen with any other extension? Or just Better Comments?

Do you have a clean install of VS? Do you have any other extensions or add-ons that you might suspect being a reason for that behavior?

**Comment by ozdemiremre** [comment]
No, only with Better Comments.

I have:
* GitHub for Visual Studio
* Productivity Power Tools 2015
* Tools for Unity
* I Hate #Regions
* MultiEditing
* Visual Studio Spell Checker

but I really cannot connect these extensions with this behavior.

Edit: I've solved the problem by changing the theme of VS to a default theme (Blue in this case) .
![123](https://cloud.githubusercontent.com/assets/8995420/16587181/069ad2e4-42d2-11e6-9527-76b62239a01a.PNG)
 I was using a custom style but when I changed the theme the problem was gone.
